app/models/survey/questions_type.rb

```diff
@@ -1,27 +1,27 @@
 class Survey::QuestionsType
-  @@questions_types = {:general => 1}
-  
+  @@questions_types = { multiple_choice: 2, free_response: 9, multi_select: 1 }
```

Comment:
Depends on what you mean by internally. Question types are required fields on question, so it's used in that sense, but the only type from before was general, so it apparently wasn't a very utilized feature